src/patterns/data/default/response_to_chemical_stimulus_trait.tsv

```diff
@@ -117,3 +117,4 @@ OBA:2050330		CHEBI:135983	teriparatide	https://orcid.org/0000-0001-6757-4744
 OBA:2050333		CHEBI:31941	oxaliplatin	https://orcid.org/0000-0001-6757-4744
 OBA:2045298		CHEBI:145810	insulin	https://orcid.org/0000-0001-8314-2140
 OBA:2045299		CHEBI:17234	glucose	https://orcid.org/0000-0001-8314-2140
+OBA:2050352		CHEBI:35526	hypoglycemic agent	https://orcid.org/0000-0001-6757-4744
```

Comment:
It would be better to use the `response_to_chemical_with_role_stimulus` pattern template, because the [CHEBI:35526 hypoglycemic agent](https://www.ebi.ac.uk/ols4/ontologies/chebi/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_35526) is a subclass of [CHEBI:50906 role](https://www.ebi.ac.uk/ols4/ontologies/chebi/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_50906?lang=en) and I would think that putting it into the `response_to_chemical_stimulus_trait` template should result in logical errors.
It is interesting that the QC did not catch it. How does `owl:Nothing` look like when you open `oba-edit.obo` in Protege?

Reply:
I think if you use the`response_to_chemical_with_role_stimulus` pattern, then the `response to hypoglycemic agent` will be a subclass of [trait in response to drug](http://purl.obolibrary.org/obo/OBA_2040016), which it is not at the moment. Using a `role` entity in the `response_to_chemical_stimulus_trait` used to generate logical errors. It is strange it does in this case.